fenix/app/onboarding.fml.yaml

```diff
@@ -0,0 +1,100 @@
+---
+features:
+
+  juno-onboarding:
```

Comment:
This is a feature, but juno is release/marketing name. Suggest a simpler `onboarding` or `onboarding-cards`.

(I haven't thought deeply on whether it's okay to just add to the existing `onboarding` feature, or not; though I expect it's okay).

Reply:
Agreed, having juno in the name is something that there were previous discussions around. We have an [existing issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1826883) for this (thanks @rahulsainani) which will cover this 👍